neo/io/proxyobjects.py

```diff
@@ -37,6 +37,17 @@ def __init__(self, array_annotations=None, **annotations):
 
         BaseNeo.__init__(self, **annotations)
 
+    def load(self, time_slice):
```

Comment:
load can have other kargs. I would put ```def load(self, time_slice, **kargs):```

Reply:
Good point. I will add that, along with `time_slice=None` since all implementations of `load` set this default.